fmf/context.py

```diff
@@ -197,15 +226,17 @@ def _op_eq(self, dimension_name, values):
         """ '=' operator """
 
         def comparator(dimension_value, it_val):
-            return dimension_value.version_cmp(it_val, ordered=False) == 0
+            return dimension_value.version_cmp(
+                it_val, ordered=False, case_sensitive=self._case_sensitive) == 0
```

Comment:
```suggestion
                it_val, ordered=False, case_sensitive=self.case_sensitive) == 0
```

Why not using the property directly?